src/Benchmarks/Data/EfDb.cs

```diff
@@ -49,9 +57,10 @@ public Task<World> LoadSingleQueryRow()
             for (var i = 0; i < count; i++)
             {
                 var id = _random.Next(1, 10001);
-                var result = await _dbContext.World.AsTracking().FirstAsync(w => w.Id == id);
+                var result = await _firstWorldQuery(_dbContext, id);
+
+                _dbContext.Attach(result).Property("RandomNumber").CurrentValue = _random.Next(1, 10001);
```

Comment:
Attach will fail any time _random.Next(1, 10001) happen to returns return the same value twice in the same call to `LoadMultipleUpdatesRows()`